.travis.yml

```diff
@@ -6,7 +6,7 @@ jdk: oraclejdk8
 before_install: echo "MAVEN_OPTS='-Xms1g -Xmx2g -XX:PermSize=512m -XX:MaxPermSize=1g'" > ~/.mavenrc
 
 install:
-  - echo 'mvn clean install -B -V 1> .build.stdout 2> .build.stderr' > .build.sh
+  - echo 'mvn clean install -B -DskipChecks=true -V 1> .build.stdout 2> .build.stderr' > .build.sh
```

Comment:
This is probably needed to the missing `about.html`?
A workaround is to add the following line to the `checkstyle.properties` file:
`checkstyle.aboutHtmlCheck.url=https://raw.githubusercontent.com/openhab/openhab2-addons/master/src/etc/NOTICE`

Without the line all local builds of bindings will still give an error on the checkstyle analysis. So I think we should do something or release the fixed sat tool asap.

Reply:
No, it is due to https://github.com/openhab/openhab2-addons/pull/4625#issuecomment-455873630.

Reply:
Note that the [Jenkins PR build](https://ci.openhab.org/job/PR-openHAB2-Addons/12073/) executed the SAT checks successfully.

Reply:
Hmm. then I have no idea why I still get that error when building locally..